packages/ui-components/style/base.css

```diff
@@ -36,13 +36,6 @@ a:hover {
   color: unset;
 }
 
-/* Disable default focus outline */
-:focus {
```

Comment:
There is a risk that this exposes the focus-visible indicator in ways that some users find ugly. But I would prefer to have an ugly focus-visible indicator that prompts aesthetic interventions from users and contributors rather than take away a key accessibility feature from users that need it.

Reply:
@gabalafou this changes the settingseditor's input:focus appearance, replacing the light blue gradient with a strong dark blue, similar to the following diff (new on left):

<img src="https://user-images.githubusercontent.com/32258950/206211246-1d0c4c54-fb1f-41c2-acdb-78f0ff18b43b.png" width="250px">

Was that expected or just a consequence ?
I'm asking this to know if I should open a PR to fix this for the settingseditor.

Reply:
No, that was not specifically intended. Sorry about that! (If you're short on time, just let me know, and I can submit a PR to fix that. I'm quite familiar now with configuring focus states in CSS, so it shouldn't take me much time.)

By the way, do you know why I cannot tab-key to the drop-downs in the settings editor?

Reply:
Thanks, I had already something about that. I opened https://github.com/jupyterlab/jupyterlab/pull/13554.

> By the way, do you know why I cannot tab-key to the drop-downs in the settings editor?

No idea, it seems to work on my end, if you mean reaching a drop-down with tab-key.